jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -57,7 +57,10 @@ public void set(String image) {
     }
   }
 
-  /** @return the {@link ImageReference} parsed from {@code from}. */
+  /**
+   * @param from the image reference string to parse.
+   * @return the {@link ImageReference} parsed from {@code from}.
+   */
   static ImageReference parseBaseImageReference(String from) {
```

Comment:
Just noticed these two methods are essentially identical. We could prob just have one method and a second parameter that's the 'parameter name'.